inventory/service/group_vars/carbonapi.yaml

```diff
@@ -10,7 +10,6 @@ carbonapi_instances:
         lbMethod: "any"
         protocol: "carbonapi_v3_pb"
         servers:
```

Comment:
Pls double check you remove the correct instance, I was sure it should be graphite2